advanced/index.md

```diff
@@ -94,14 +94,14 @@ $ git log -G [regex-pattern-in-patch]
 $ git log --grep=[regex-in-message]
 ```
 
-Filtering by file state, *added* (A), *modified* (M), or *deleted* (D), also narrows what change requires assessment.
+Filtering by file state, i.e. *added* (A), *modified* (M), or *deleted* (D):
```

Comment:
Perhaps we can add back in what/why we would want to filter by state, so that the sentence gives some context/reason to the action.

Would this not be a case for "e.g." as they are options to choose between, but not actions to follow as-is?